src/main/java/com/cupid/jikting/common/filter/ExceptionHandlerFilter.java

```diff
@@ -0,0 +1,48 @@
+package com.cupid.jikting.common.filter;
+
+import com.cupid.jikting.common.dto.ErrorResponse;
+import com.cupid.jikting.common.error.ApplicationException;
+import com.cupid.jikting.common.error.ExpiredJwtException;
+import com.cupid.jikting.common.error.InvalidJwtException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ExceptionHandlerFilter extends OncePerRequestFilter {
+
+    private static final String ENCODING = "UTF-8";
+
+    @Override
+    protected void doFilterInternal(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            FilterChain filterChain
+    ) throws ServletException, IOException {
```

Comment:
p2. 메소드 선언이 길어질 때는 120자를 넘어가는 파라미터 앞에 개행해주세요!

아래 코드는 들여쓰기가 안맞을 수도 있어요!

```suggestion
    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
            FilterChain filterChain) throws ServletException, IOException {
```

Reply:
이 코멘트에 대해서는 반영이 안된건가요, 원래처럼 하는게 더 좋다고 생각하시는건가요?